superset/assets/src/visualizations/BigNumber/BigNumber.jsx

```diff
@@ -26,7 +23,7 @@ const PROPORTION = {
   TRENDLINE: 0.3,
 };
 
-function renderTooltipFactory(formatValue) {
+export function renderTooltipFactory(formatValue) {
```

Comment:
This was used only in this file. Not sure it is useful to export now. (ok to expert if there are other places using or unit test)

Reply:
Actually it is also used in adaptor for the prop `renderTooltip`. That is why I exported it :)

Reply:
I missed that. Thank you!